docker/docker-compose-tests.yml

```diff
@@ -6,6 +6,4 @@ services:
     container_name: tests
     build:
       target: build
```

Comment:
What are we using this file for?

Reply:
Now removed as a part of this PR

Reply:
Believe this file will be used once the pipeline is built, to change the entry point to gradle check to run all tests.